src/Raven.Embedded/ServerOptions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+
+namespace Raven.Embedded
+{
+    public class ServerOptions
+    {
+        public string FrameworkVersion { get; set; } = "2.1.0";
+
+        public string DataDir { get; set; }
+
+        public List<string> CommandLineArgs { get; set; } = new List<string>
+        {
+            "--ServerUrl=http://127.0.0.1:0",
+            "--RunInMemory=false",
```

Comment:
Is this necessary? Let people choose that.

Reply:
They can it is just the default